README.md

```diff
@@ -312,6 +312,38 @@ The following sample plugins exist for NRI:
 Please see the documentation of these plugins for further details
 about what and how each of these plugins can be used for.
 
+## Security Considerations
+
+From a security perspective NRI plugins should be considered part of the
+container runtime. NRI does not implement granular access control to the
+functionality it offers. Access to NRI is controlled by restricting access
+to the systemwide NRI socket. If a process can connect to the NRI socket
+and send data, it has access to the full scope of functionality available
+via NRI.
+
+The same precautions and principles apply to protecting the NRI socket as
+to protecting the socket of the runtime itself. Unless it already exists,
```

Comment:
Is there an existing list of these precautions we provide for the runtime socket that we could link to or summarize here? Or can we be more explicit about how we recommend protecting and restricting access to the socket? 
e.g. do not change the default file permissions.

Reply:
> Is there an existing list of these precautions we provide for the runtime socket that we could link to or summarize here?

No, at least based on quick git grep I could not find anything relevant in the in-tree containerd documentation.

> Or can we be more explicit about how we recommend protecting and restricting access to the socket? e.g. do not change the default file permissions.

So, should we explicitly discourage changing the socket permissions ? State something like 

```
Changing the default socket permissions is strongly advised against. Enabling more permissive access control to NRI should never be done without fully understanding the full implications and potential consequences to container security.
```

Reply:
That wording looks good to me.